### DISCUSSION TITLE
Torchmetrics vs pytorch_lightning.metrics

### DISCUSSION THREAD

**Discussion by mmg10** [Lightning-AI/torchmetrics#82] [Q&A] [answered]
Hi, Can you please tell what is the difference between Torchmetrics and pytorch_lightning.metrics? Both seem to be the same (at least from a functional point-of-view)

**Comment by hassiahk** [comment, marked as answer]
They are the same but the `pytorch_lightning.metrics` will soon be deprecated as they are migrating all the codebase from `pytorch_lightning.metrics` to `torchmetrics`.

**Reply by Borda** [reply]
yes, we started the deprecation now and aiming at v1.3 for having a full re-route (`pytorch_lightning.metrics` will be `torchmetrics` under the hood) and remove `pytorch_lightning.metrics` in v1.5
